rotkehlchen/tests/fixtures/thegraph.py

```diff
@@ -0,0 +1,16 @@
+from typing import TYPE_CHECKING
+import pytest
+
+from rotkehlchen.types import ApiKey, ExternalService, ExternalServiceApiCredentials
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+@pytest.fixture(name='add_subgraph_api_key')
+def fixture_add_subgraph_api_key(database: 'DBHandler') -> None:  # noqa: PT004 # adding _ won't export it
+    """ count generator used to get a unique port number. """
+    database.add_external_service_credentials([ExternalServiceApiCredentials(
+        service=ExternalService.THEGRAPH,
+        api_key=ApiKey('d943ea1af415001154223fdf46b6f193'),
```

Comment:
the key is repeated here and in test_ethereum_transactions.py. Make it into a constant and put it in test constants and use it in both places

Reply:
I wanted to avoid putting it into a constant to avoid scripts scrapping github 😓

Reply:
It's already scraped as a variable called api key ;)